src/style-library/components/pages/ImageScreen.scss

```
@@ -37,13 +37,27 @@
+// something that reads as "the image", an inset frame is just a smaller rectangle and the band around
+// it does not obviously belong to the background.
 .kadence-blocks-style-library__image-preset-preview-photo {
-	display: block;
+	display: flex;
+	align-items: center;
+	justify-content: center;
 	width: 100%;
 	height: 100%;
 	border-radius: inherit;
 	background: var(--kb-sl-color-gray-700);
+	// Sized in `em` off a font-size floor rather than a fixed px, so the glyph shrinks with the photo
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)